AEPTarget/Sources/Event+Target.swift

```diff
@@ -68,4 +68,9 @@ extension Event {
     var isClearPrefetchCache: Bool {
         data?[TargetConstants.EventDataKeys.CLEAR_PREFETCH_CACHE] as? Bool ?? false
     }
+
+    /// Returns error message in the response event
+    var error: String {
```

Comment:
i guess if we expect there to be no error for successful requests, it would be better to make this a `String?` and just return `nil` when no error is present in the data.  that cleans up usage:

```
if let error = event.error {

vs

if let error = event.error, !error.isEmpty {
```